libsoftwarecontainer/src/gateway/devicenode/devicenodegateway.cpp

```diff
@@ -31,36 +31,35 @@ DeviceNodeGateway::DeviceNodeGateway() :
 {
 }
 
-ReturnCode DeviceNodeGateway::readConfigElement(const json_t *element)
+bool DeviceNodeGateway::readConfigElement(const json_t *element)
 {
     DeviceNodeParser parser;
     DeviceNodeParser::Device dev;
 
-    if (isError(parser.parseDeviceNodeGatewayConfiguration(element, dev))) {
+    if (!parser.parseDeviceNodeGatewayConfiguration(element, dev)) {
         log_error() << "Could not parse device node configuration";
-        return ReturnCode::FAILURE;
+        return false;
     }
 
     return m_logic.updateDeviceList(dev);
 }
 
 bool DeviceNodeGateway::activateGateway()
 {
-    auto returnValue = applySettings();
-    return isSuccess(returnValue);
+    return applySettings();
```

Comment:
Is there a good reason to still keep `applySettings()` instead of moving its code here?